packages/components/src/text/Text.styles.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Nit:

All filenames in Gutenberg follow [kebab case style](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles) or at least they should. @youknowriad or @aduth - do you know if we have it covered in docs? Is it WordPress specific maybe?